services/ui-src/src/utils/autosave/helpers.ts

```diff
@@ -0,0 +1,50 @@
+import { FieldValues, UseFormReturn } from "react-hook-form";
+import { Choice, DropdownChoice, EntityShape, ReportStatus } from "types";
+
+const valueChanged = (currentValue: any, storedValue: any) => {
+  return currentValue !== storedValue;
+};
```

Comment:
Nit: Probably don't need this method since it's just a single inequality check. In general we don't want to get tooo small with our utilities when they're easy things.